README.md

```diff
@@ -15,20 +15,12 @@ of popular platforms like Codeforces, Codechef, TopCoder etc.
 
 ## Quick start
 
-1. [Install cph](https://marketplace.visualstudio.com/items?itemName=DivyanshuAgrawal.competitive-programming-helper)
-   in VS Code and open any folder.
-
-1. [Install competitive companion](https://github.com/jmerle/competitive-companion#readme)
-   in your browser.
-
-1. Use Companion by pressing the green plus (+) circle from the browser toolbar
-   when visiting any problem page.
-
-1. The file opens in VS Code with testcases preloaded. Press `Ctrl+Alt+B` to run
-   them.
-
-1. Install the [cph-submit](https://github.com/agrawal-d/cph-submit) browser
-   extension to enable submitting directly.
+1. [Install cph](https://marketplace.visualstudio.com/items?itemName=DivyanshuAgrawal.competitive-programming-helper) in VS Code and open any folder.
+1. [Install competitive companion](https://github.com/jmerle/competitive-companion#readme) in your browser.
+1. Use Companion by pressing the green plus (+) circle from the browser toolbar when visiting any problem page.
+1. The file opens in VS Code with testcases preloaded. Press `Ctrl+Alt+B` to run them.
+1. (Optional) Install the [cph-submit](https://github.com/agrawal-d/cph-submit) browser extension to enable submitting directly on CodeForces.
+1. (Optional) Install submit client and config file from the [Kattis help page](https://open.kattis.com/help/submit) after logging in.
```

Comment:
Remove these two lines from this list, they can appear as an unordered list just below this list.